### PR TITLE
Changes in the image for test_load_from_url

### PR DESCRIPTION
**Pull request by SidharthBansal**
## PR Summary

Image for the test_load_from_url changed so that it can use the images which are not baseline images. This will be later benificial for the division of mpl into mpl and mpl_baseline_images package as #17557 

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/tests/test_image.py [hunk 1]
```diff
@@ -713,7 +713,7 @@ def test_minimized_rasterized():
 
 
 def test_load_from_url():
-    path = Path(__file__).parent / "baseline_images/test_image/imshow.png"
+    path = Path(__file__).parent / "baseline_images/pngsuite/basn3p04.png"
```
Comment (jklymak) [inline lib/matplotlib/tests/test_image.py:716]: I don't understand why we think this image will be permanent either, and it seems a very obscure image to choose.  Isn't there something in mpl_data/images that would be more appropriate?

Comment (SidharthBansal) [reply lib/matplotlib/tests/test_image.py:716]: This image will remain as it is also used in https://github.com/matplotlib/matplotlib/blob/50a535c2453f438a5bfbfc3bc7275c8dafc2f422/lib/matplotlib/tests/test_image.py#L151. 
Secondly, the `basn3p04.png` image is not used in `image_comparison` tests. So, we don't need to shift it into the `mpl_baseline_images` package in #17557

Comment (jklymak) [reply lib/matplotlib/tests/test_image.py:716]: But shouldn’t that be moved as well?  It’s going to be pretty confusing to have baseline_images and another package of baseline_images.

Comment (tacaswell) [reply lib/matplotlib/tests/test_image.py:716]: The primary reason the `basn*.png` files exist is as input for the test in `test_png` so is not in the category of images that will be removed by the GSOC code landing.  On the other hand, the file that this used to point is a baseline image that will be removed.

**Pull request merged**